### PR TITLE
Refactor integrate stage 2

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: base/commands/list/list_set.go [hunk 1]
```diff
@@ -6,54 +6,65 @@ import (
...
+		}
+		data := codec.DecodeListSetResponse(resp)
+		return convertDataToRow(ci, data, ec.Props().GetBool(base.FlagShowType))
 	})
 	if err != nil {
 		return err
 	}
 	stop()
-	return nil
+	return ec.AddOutputRows(ctx, rowV.(output.Row))
```
Comment (mtyazici) [inline base/commands/list/list_set.go:67]: We are printing the last set value as follows
```
Value 
last-val  
```
We are also using the same title for every return type. I find this a bit confusing, what do you think?

Comment (yuce) [reply base/commands/list/list_set.go:67]: > same title for every return type

Do you mean every data structure?

Comment (mtyazici) [reply base/commands/list/list_set.go:67]: Not every data structure, I realized this function `convertDataToRow(ci *hazelcast.ClientInternal, data hazelcast.Data, showType bool)` is only used once in `list set`. 

What I meant is we are printing the correct title for every other command and for this one we are printing the last value with the `Value` title which is a bit misleading.

Comment (yuce) [reply base/commands/list/list_set.go:67]: Changed the titles at:
* https://github.com/hazelcast/hazelcast-commandline-client/pull/372/commits/e9cfedad237d7557064057f20e20803dad14f0c9
*  https://github.com/hazelcast/hazelcast-commandline-client/pull/372/commits/9965a3088c4445fbc02d47e620525d8a0eb320c3

**Review by mtyazici — APPROVED** [review]

**Pull request merged**